log4j-core/src/main/java/org/apache/logging/log4j/core/pattern/ThrowableStackTraceRenderer.java

```diff
@@ -268,9 +268,15 @@ static final class Metadata {
              */
             final int stackLength;
 
-            private Metadata(final int commonElementCount, final int stackLength) {
+            /**
+             * The suppressed exceptions attached to this {@link Throwable}
```

Comment:
```suggestion
             * The suppressed exceptions attached to this {@link Throwable}.
             * This needs to be captured separately since {@link Throwable#getSuppressed()} can change.
             * 
             * @see <a href="https://github.com/apache/logging-log4j2/issues/3929">#3929</a>
             * @see <a href="https://github.com/apache/logging-log4j2/pull/3934">#3934</a>
```